TYPO3.Flow/Classes/TYPO3/Flow/Resource/ResourceRepository.php

```diff
@@ -106,6 +122,62 @@ public function findByIdentifier($identifier)
     }
 
     /**
+     * Iterator over an IterableResult and return a Generator
+     *
+     * This methos is useful for batch processing huge result set as it clear the object
+     * manager and detach the current object on each iteration.
```

Comment:
This is a false promise, no? The code here doesn't do any of that! It just makes it possible…